access-key.go

```diff
@@ -44,10 +44,10 @@ const (
 )
 
 // isValidSecretKey - validate secret key.
-var isValidSecretKey = regexp.MustCompile("^.{40}$")
+var isValidSecretKey = regexp.MustCompile("^.{8,40}$")
 
 // isValidAccessKey - validate access key.
-var isValidAccessKey = regexp.MustCompile("^[A-Z0-9\\-\\.\\_\\~]{20}$")
+var isValidAccessKey = regexp.MustCompile("^[a-zA-Z0-9\\-\\.\\_\\~]{5,20}$")
 
 // mustGenAccessKeys - must generate access credentials.
 func mustGenAccessKeys() (creds credential) {
```

Comment:
UI does not have the `show current password` feature yet, i will tag a new release after that.